src/js/components/Header.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+
+class Header extends React.Component {
```

Comment:
Hi Chris! I'm testing Github right now :)

On Wed, May 4, 2016 at 12:03 AM, chris⚡ <notifications@github.com> wrote:

> In src/js/components/Header.js
> <https://github.com/dcos/dcos-ui/pull/82#discussion_r61996019>:
>
> > @@ -0,0 +1,80 @@
> > +import React from 'react';
> > +
> > +class Header extends React.Component {
>
> done!
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/dcos/dcos-ui/pull/82/files/45908c02b13614f82ca5d3032b1f741bd85f9747#r61996019>
>